tools/SetupFlow/DevHome.SetupFlow/ViewModels/AddRepoViewModel.cs

```diff
@@ -1401,6 +1410,16 @@ private async Task CoordinateTasks(string loginId, Task<RepositorySearchInformat
             SelectionOptionsPlaceholderText = repoSearchInformation.SelectionOptionsPlaceHolderText;
 
             IsFetchingRepos = false;
+
+            if (!_repositoriesForAccount.Any())
+            {
+                NoRepositoriesMessage = _stringResource.GetLocalized(StringResourceKey.RepoToolNoRepositoriesMessage, _providers.DisplayName(_selectedRepoProvider));

```

Comment:
>_stringResource.GetLocalized(StringResourceKey.RepoToolNoRepositoriesMessage, _providers.DisplayName(_selectedRepoProvider)); [](http://example.com/codeflow?start=40&length=125)

How come you need to load it yourself?  Can't the xaml just bind to the value in resw directly?

Reply:
You are correct.  I need to bind it here because of the wildcard.  If XAML binded to this string.  The user would see "Added repositories are found in your {0} account"

The replacement need to be done in code.